test/stateful.sh

```diff
@@ -55,7 +55,7 @@ job-control() {
 }
 
 bind() {
-  spec/stateful/bind.py $FIRST --oils-failures-allowed 2 "$@"
+  spec/stateful/bind.py $FIRST --oils-failures-allowed 4 "$@"
```

Comment:
Hm we are not passing `--num-lines` or `--num-columns` here?

Another interface we can consider is to do it per test case, like `@register(num_lines=50, num_cols=80)`

Not sure if that's better though

Reply:
Heh, I should have put it in. The defaults are fine, but pyte-based tests should probably specify the exact dims.

If nothing else, it communicates that the dims are actually important there.